src/main/webapp/placeGuides/PlaceGuideManager.js

```diff
@@ -1,32 +1,61 @@
 /**
  * This class is responsible for connecting all the placeGuide-related
  * objects: the two displayers and the repository.
- * When an event occurs, PlaceGuideManager is triggered and it
- * sends the commands to the
- * classes which are responsible for handling the events.
+ * When the map's bounds are changed, PlaceGuideManager
+ * sends the eneded commands to all classes involved in
+ * executing it(PlaceGudieRepository, Map- and ListPlaceGuideDisplayer).
  */
 class PlaceGuideManager {
-  constructor(placeGuideRepository) {
-    this._placeGuideRepository = placeGuideRepository;
+  static PAGE = {
+    DISCOVER: {
+      query: PlaceGuideRepository.QUERY_TYPE.ALL_PUBLIC_IN_MAP_AREA,
+    },
+    MY_GUIDES: {
+      query: PlaceGuideRepository.QUERY_TYPE.CREATED_ALL_IN_MAP_AREA,
+    },
+    CREATE_PLACE_GUIDE: {
+      query: PlaceGuideRepository.QUERY_TYPE.CREATED_ALL_IN_MAP_AREA,
+    },
+    BOOKMARKED_PLACEGUIDES: {
+      query: PlaceGuideRepository.QUERY_TYPE.BOOKMARKED,
+    }
+  };
+
+  constructor(page, map) {
+    this._page = page;
+    this._placeGuideRepository = new PlaceGuideRepository();
     this._highlightedPlaceGuideId = null;
     this._mapPlaceGuideDisplayer = new MapPlaceGuideDisplayer();
     this._listPlaceGuideDisplayer = new ListPlaceGuideDisplayer();
```

Comment:
so we should pass the query type as an argument for the constructor right?

Reply:
Yes, we will, as soon as we'll have the updated ListPlaceGuideDisplayer.